__tests__/models.test.js

```diff
@@ -82,6 +84,77 @@ describe("2nd endpoint, reviews, can return status 200", () => {
     })
 })
 
+describe("3rd endpoint, parametric. Get reviews by I.D", () => {
+    test("end point response with status 200", () => {
+        return request(app).get('/api/reviews/1').expect(200)
+    })
+    test("The returned reviews object contains the correct categories", () => {
+        return request(app).get('/api/reviews/1').expect(200)
+        .then(response => {
+            const review = response.body
+                expect(review[0]).toHaveProperty('owner', expect.any(String))
+                expect(review[0]).toHaveProperty('title', expect.any(String))
+                expect(review[0]).toHaveProperty('votes', expect.any(Number))
+                expect(review[0]).toHaveProperty('created_at', expect.any(String))
+                expect(review[0]).toHaveProperty('category', expect.any(String))
+                expect(review[0]).toHaveProperty('review_img_url', expect.any(String))
+                expect(review[0]).toHaveProperty('designer', expect.any(String))
+                expect(review[0]).toHaveProperty('review_id', expect.any(Number))
+        
+        })
+    })
+    test("The response body only contains one element, which has the correct ID", () => {
+        return request(app).get('/api/reviews/1').expect(200)
+        .then(response => {
+            const review = response.body
+            expect(review.length).toBe(1)
+            expect(review[0].review_id).toBe(1)
+        })
+    }) 
+    test("Will respond with an error message if an unavailable ID is given", () => {
+        return request(app).get('/api/reviews/10000').expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe("Not Found")
+        })
+    })
+})
+describe("4th endpoint, comments by review id", () => {
+    test("4th endpoint responds with a status 200", () => {
+        return request(app).get('/api/reviews/2/comments').expect(200)
+    })
+    test("The return object has the correct keys for a comment", () => {
+        return request(app).get('/api/reviews/2/comments').expect(200)
+        .then (response => {
+            const comments = response.body
+            comments.forEach((comment) => {
```

Comment:
please add an assertion for the length before running this forEach() - this is important as if the returned rows were empty, the forEach would not be executed.